### DISCUSSION TITLE
如果我的臉或圖「被盜上架」成其他人的 NFT，法律上怎麼辦?

### DISCUSSION THREAD

**Discussion by JAIMEHUNG** [1101-GS4719-C/Python-Community-GS4719-C#288] [General]
這篇文章有題及NFT作品涉及著作人格權與肖像權，若有人盜用你的臉做成NFT上架，我們有何法律途徑?
因為擁有NFT頂多是擁有其「冠名權」而非所有權，所以民法的部分很難觸及到。
我認為平台方可以想出對應的機制，法律的部分也可以有所跟進。
https://www.inside.com.tw/article/26249-legal-issues-of-nft

**OpenSea 宣布將禁止「盜版」無聊猿 NFT販售:**
https://www.inside.com.tw/article/26226-opensea-bans-phayc-phunky-ape-yacht
我目前有看到是平台 OpenSea 宣佈將會禁止抄襲的作品在上面販售，大家有任何想法都歡迎補充！

**Comment by 107602505** [comment]
NFT現在很熱門，且現有法律針對 NFT 數位市場關於智慧財產權相關規仍然不足，這也的確是各國政府目前法令難以界定的範疇，記得在前幾篇有看到我國政府即將設立數位發展部，或許不久的將來就會設立跟NFT有關的法規了。

**Comment by ghost** [comment]
摁的確如果一大堆盜版品在虛擬世界上交易一定會引發很大的問題，像是如何確認NFT畫作的所有權就是個問題，相似的畫舫畫能不能也拿來販售等等

**Comment by ChinChun0220** [comment]
那不知道抄襲、模仿、二創這種圖該怎麼定義呢跟處理呢？

**Comment by cody910120** [comment]
我只能說真正值錢的只有圖片的原檔，其他任何的盜版跟抄襲，只是拿來滿足虛心而已

**Comment by luo910227** [comment]
NFT確實需要適當的管控單位去監督最好也仍能連結於現實法律，不然持續發展可能會有更多比原先網際網路更嚴重的問題。

**Comment by Joan0518** [comment]
不知道opensea要如何杜絕盜版上架的問題?

**Comment by chupochun** [comment]
的確，如果真的遇到這種問題，到底應該怎麼解決呢

**Comment by xkllkx** [comment]
希望有適合的法律能夠控管這種事情，但在區塊鍊去中心化的架構下真的抓的到人嗎？
